InvisibleApi/Extensions/InvisibleApiApplicationBuilderExtension.cs

```diff
@@ -4,18 +4,23 @@
 // See License.txt in the project root for license information.
 // ---------------------------------------------------------------
 
-using System.Collections.Generic;
 using InvisibleApi.Middlewares;
 using InvisibleApi.Models.InvisibleApiConfigurations;
+using InvisibleApi.Models.InvisibleApiProfiles;
 using Microsoft.AspNetCore.Builder;
+using System.Collections.Generic;
 
 namespace InvisibleApi.Extensions
 {
     public static class InvisibleApiApplicationBuilderExtension
     {
         public static IApplicationBuilder UseInvisibleApis(
-            this IApplicationBuilder app, 
-            List<InvisibleApiConfiguration> invisibleApiDetails) =>
-                app.UseMiddleware<InvisibleApiMiddleware>(invisibleApiDetails);
+            this IApplicationBuilder app,
+            List<InvisibleApiConfiguration> invisibleApiDetails,
+            List<InvisibleApiProfile> invisibleApiProfiles)
+        {
+            object[] parameters = { invisibleApiDetails, invisibleApiProfiles };
+            return app.UseMiddleware<InvisibleApiMiddleware>(parameters);
```

Comment:
put an empty line between your return statement and all the above code.

Reply:
I am not used to forking and merging code like this. I didn't realize that all the changes I made were going to get included in the pull request. :| Thanks.